### PR TITLE
tetragon: Add new NoPost and Signal actions

### PR DESCRIPTION
**Pull request by olsajiri**
Adding new nopost and signal actions together with fixes to allow
3 actions for selector.

New Signal action sends configured signal (argSig) to the current task:
```
      matchActions:
      - action: Signal
        argSig: 10
```

New NoPost action prevents the event to be sent to user space:
```
      matchActions:
      - action: Override
        argError: -2
      - action: NoPost
```

### CODE REVIEW

**Comment by mtardy** [comment]
Related to https://github.com/cilium/tetragon/issues/730

File: bpf/process/types/basic.h [hunk 1]
```diff
@@ -73,6 +73,7 @@ enum {
 	ACTION_COPYFD = 5,
 	ACTION_GETURL = 6,
 	ACTION_DNSLOOKUP = 7,
+	ACTION_NOPOST = 8,
```
Comment (tixxdz) [inline bpf/process/types/basic.h:76]: Hey @olsajiri so we won't get the override event?

File: pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml [hunk 2]
```diff
@@ -192,6 +192,10 @@ spec:
                                     action
                                   format: int32
                                   type: integer
+                                argSig:
+                                  description: A signal number for signal action
+                                  format: int32
```
Comment (tixxdz) [inline pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml:197]: Can we make please a string here? so we can handle both "10" or "SIGUSR1" as an example; all of them start with "SIG*"

**Review by kkourt — APPROVED** [review]
Thanks, LGTM!

File: bpf/process/types/basic.h (hunk 1 above)
Comment (olsajiri) [reply bpf/process/types/basic.h:76]: hum, not sure what you mean, there's OVERRIDE enum earlier in the list

File: pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml (hunk 2 above)
Comment (olsajiri) [reply pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml:197]: good idea, is that ok as follow up change?

Comment (kkourt) [reply pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml:197]: I also think suggestion make sense.
Either option (doing the in this PR or as a followup) is fine with me.
Also, found the the function we can use: https://pkg.go.dev/golang.org/x/sys/unix#SignalNum.

**Review by tixxdz — APPROVED** [review]
Thank you!

**Pull request merged**